test/Microsoft.Azure.Devices.Edge.Test/Provisioning.cs

```diff
@@ -37,6 +37,7 @@ string DeriveDeviceKey(byte[] groupKey, string registrationId)
 
         [Test]
         [Category("CentOsSafe")]
+        [Category("Unstable")]
```

Comment:
Do we have any associated task to stabilize these? If so, is it worth having a small comment with the task number?